dashboard/config/scripts/netsim.script

```diff
@@ -3,5 +3,19 @@ description_short ""
 description ""
 hidden 'true'
 
+stage 'Sending Bits'
+level 'blockly:NetSim:unit1_lesson04'
```

Comment:
Hm... should I be using .level files, or just a new DSL (`.netsim` files, I suppose)?